src/utils/https.service.js

```diff
@@ -1,20 +1,63 @@
-export class httpsService {
+import axios from 'axios';
+import { config as conf } from '../config/conf';
+import { storageService } from './storage.service';
 
-    get () {
-    
+class HttpsService {
+    get(url, options) {
+        return this.makeRequest({
+            url,
+            method: 'GET',
+            ...options,
+        })
     }
-    post () {
-    
-    }
-    put () {
-    
+
+    post(url, body, options) {
+        return this.makeRequest({
+            url,
+            data: body,
+            method: 'POST',
+            ...options,
+        })
+    }   
+
+    put(url, body, options) {
+        return this.makeRequest({
+            url,
+            data: body,
+            method: 'PUT',
+            ...options,
+        })
     }
-    delete () {
-    
+
+    delete(url, body, options) {
+        return this.makeRequest({
+            url,
+            data: body,
+            method: 'DELETE',
+            // headers: '';
+            ...options,
+        })
     }
-    makeRequest () {
-    
+
+    makeRequest(config) {
+        //config.headers['Content-Type'] = 'application/json';
```

Comment:
why do you comment this?

Reply:
Cos it throws the following error 'Cannot set properties of undefined (setting 'Content-Type')' and for now I don't know how to handle this.

Reply:
deleted it. we have default header: Content-Type: application/json; charset=utf-8